srcpkgs/ananicy/files/ananicy/log/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec logger -t ananicy
```

Comment:
This should use `vlogger(1)` instead of logger.

Reply:
Was based on another package as an example on how to use the logger, so I used the same one:

https://github.com/void-linux/void-packages/blob/master/srcpkgs/earlyoom/files/earlyoom/log/run

Reply:
Done, btw :)

Reply:
this was not fixed